workshops/py_Christmas/README.md

```diff
@@ -0,0 +1,381 @@
+
+speed(0)
+
+bg = turtle.Screen()
+bg.bgcolor("#99004D") #Setting back ground color
+
+y = -100
+width = 240
+
+# Constructing the tree
+box1(turtle, "#4F4553", -15, y - 40, 30, 40) #Stem of the tree
+
+
+# Tree
+while width > 20 :
+    width = width - 20
+    height = randint(20,30)
+    x = 0 - width/2
+    box1(turtle, "#05A167", x, y, width, height) # Box for the leaves of tree
+    balls(turtle, "#fc045b", x, y, 5) # Decorative balls
+    balls(turtle, "#fc045b", -x, y, 5) # Decorative balls
+    y = y + height
+
+#Star on the top 
+star(turtle, "#e5f614", 0.4, y, 20)
+
+#Christmas wishes
+penup()
+color("#ff59ac")
+goto(-250, -250)
+write("Merry Christmas !!!", font=("Calibri", 50, "bold"))
+
+hideturtle()
+
+turtle.done()
+
+```
+
+Now lets run the code . 
+
+# Running the code
+Click the green "Run" button on the top of the repl.it window. A screen will be shown on the right side displaying the turtle graphics that you coded. 
+If you face any errors, try commenting out each portion of the code and making only a certain section work. This generally helps to figure out the error.
+![alt text](https://cloud-4ipo9natc.vercel.app/2image9.jpeg) 
+# One last thing
+
+   Did you find this fun ? If yes, try out some of your own designs. 
+You can basically draw almost anything with this. 
+A few suggestions would be:
+1. Try changing the ******speed()****** function to '1' (or other values) to see the difference in speed. Try changing color of the parameters in the code. You can use [this link](https://www.color-hex.com/) to find the right hexcode you want.
+2. Try adding some more additions into this code,like maybe a few giftboxes on the bottom of the tree. For this you could use the same rectangle function. 
+You could also try adding a few more decorative stuff into the tree or the back ground
+3. Try making an entire different design. Maybe a pet animal or a house or literally anything you wish. The pic given below is something we call "pookkalam". If you google search this word, you will get to know that these are designs we (South Indians) make on the floor using flower petals. A friend of mine coded a similar design using turtle. A bit complicated, but superfun.
+![alt text](https://github.com/FossMec/Code-a-pookalam/blob/master/Nikita%20Menon/FinalPookalam.png)
+[Code for the "pookkalam" made using turtle](https://github.com/FossMec/Code-a-pookalam/tree/master/Nikita%20Menon)
+4. Now, I would suggest to try out some other turtle functions that you may find interesting.    
+   [This link](https://runestone.academy/runestone/books/published/StudentCSP/CSPTurtleDecisions/turtleFP.html) provides details of most functions available in turtle.
+
+Aight ! Congrats on completing the workshop. Will you do me a small favour now ? Next time you try out a design, dm me the output in twitter (@raz8153) or on hack club clack channel (@Razz).
+I would Love to see your work.
+
+****Thanks A Lot for sticking till the end and Happy hacking !!!**** 
+
```

Comment:
```suggestion
# One last thing

   Did you find this fun? If yes, try out some of your own designs. 
You can basically draw almost anything with this. 
A few suggestions would be:
1. Try changing the ******speed()****** function to '1' (or other values) to see the difference in speed. Try changing the color of the parameters in the code. You can use [this link](https://www.color-hex.com/) to find the right hex code you want.
2. Try adding some more additions to this code, like maybe a few gift boxes on the bottom of the tree. For this, you could use the same rectangle function. 
You could also try adding a few more decorative stuff into the tree or the background
3. Try making entirely different designs. Maybe a pet animal or a house or literally anything you wish. The pic given below is something we call "pookkalam". If you google search this word, you will get to know that these are designs we (South Indians) make on the floor using flower petals. A friend of mine coded a similar design using a turtle. A bit complicated, but super fun.
![alt text](https://github.com/FossMec/Code-a-pookalam/blob/master/Nikita%20Menon/FinalPookalam.png)
[Code for the "pookkalam" made using turtle](https://github.com/FossMec/Code-a-pookalam/tree/master/Nikita%20Menon)
4. Now, I would suggest to try out some other turtle functions that you may find interesting.    
   [This link](https://runestone.academy/runestone/books/published/StudentCSP/CSPTurtleDecisions/turtleFP.html) provides details of most functions available in turtle.

Aight ! Congrats on completing the workshop. Will you do me a small favour now ? Next time you try out a design, dm me the output in twitter (@raz8153) or on hack club clack channel (@Razz).
I would Love to see your work.

****Thanks A Lot for sticking till the end and Happy hacking !!!**** 

```

Reply:
Find the English word for that Malayalam word.

Reply:
> 
> 
> Are you an active slack member or hack club lead?
Hack Club MEC lead . Slack id : Razz

Reply:
Hey, Thanks a lot for giving such specific suggestions. Sorry about the mistakes. Will fix them asap